ui/admin/tests/acceptance/roles/read-test.js

```diff
@@ -82,4 +82,15 @@ module('Acceptance | roles | read', function (hooks) {
     await visit(urls.roles);
     assert.notOk(find('main tbody .hds-table__tr:nth-child(1) a'));
   });
+
+  test('can navigate to a role and fetches scopes correctly', async function (assert) {
+    assert.expect(1);
+    await visit(urls.roles);
+    await click('main tbody .hds-table__tr:nth-child(1) a');
+    await a11yAudit();
+    const { type: type } = this.server.schema.roles.findBy({
```

Comment:
Does this assert that project scopes doesn't get queried? I think the easiest solution is to just setup your server to return an error if your previously created project ID gets called. Before the bug fix this would cause the test to fail but should pass with the fix